website/docs/rules/common/ban-name.md

```diff
@@ -0,0 +1,43 @@
+# Ban name
+
+## Rule id {#rule-id}
+
+ban-name
+
+## Severity {#severity}
+
+Warning
+
+## Description {#description}
+
+Configure some names that you want to ban.
+
+For example, in my own app, I want to ban `showDialog` and force to use `myShowDialog` instead. This is because (let me explain the simplified case) I added logging in `myShowDialog`, and I do want every dialog to be logged.
```

Comment:
Could you rephrase this description to a more abstract form? I totally get the idea and the example is legit, but it'd be better to have `you` as pointing to the user standpoint, not `I`.